Octokit/Helpers/ApiUrls.cs

```diff
@@ -1153,6 +1153,18 @@ public static Uri CreateTag(string owner, string name)
             return "repos/{0}/{1}/git/tags".FormatUri(owner, name);
         }
 
+        /// <summary>
+        /// Returns the <see cref="Uri"/> for creating the event.
+        /// </summary>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="repo">The name of the repository</param>
+        /// <param name="workflowId">The ID of the workflow</param>
+        /// <returns></returns>
+        public static Uri CreateWorkflowDispatchEvent(string owner, string repo, string workflowId)
+        {
+            return "repos/{owner}/{repo}/actions/workflows/{workflow_id}/dispatches".FormatUri(owner, repo, workflowId);
```

Comment:
This uses named holes (is that even supported?), while the other method uses indexed holes. Also, there are a bunch of inconsistencies in `name` vs `repo` across the board. I suggest using `repo` throughout 😊

Reply:
Thank you for such quick feedback. :smile: I will fix `name` vs `repo`. I have not tested yet if those holes are supported. I got inspired from 2 different (and inconsistent Client implementations). I am not so familiar with React framework and I was hoping to get some feedback there. Furthermore I am a bit limited in testing workflow dispatch or `GetAllForRepository` workflows. :smile: